angular.json

```diff
@@ -29,7 +29,8 @@
             "assets": [
               "src/favicon.ico",
               "src/assets",
-              "src/manifest.webmanifest"
+              "src/manifest.webmanifest",
+              "src/robots.txt"
```

Comment:
we only want to have robots available with ssr. No assets needed. Just put it in project root.